src/app/pages/organizations/onc-organizations/onc-organizations.jsx

```diff
@@ -199,30 +201,26 @@ function ChplOncOrganizations() {
                 )}
               { isEditing !== 'org' && orgType === 'acb'
                 && (
-                  <ChplUsers
-                    users={users}
-                    roles={roles}
-                    dispatch={handleDispatch}
-                  />
+                  <ChplUsers users={users} roles={roles} dispatch={handleDispatch} />
```

Comment:
Once it gets to two or three attributes, it's usually best to break them into multiple lines. I don't think this change is one we should keep